predicators/utils.py

```diff
@@ -3384,3 +3384,29 @@ def get_task_seed(train_or_test: str, task_idx: int) -> int:
     # Need to cast to int because generate_state() returns a numpy int.
     task_seed = int(seed_sequence.generate_state(task_idx + 1)[-1])
     return task_seed
+
+
+def beta_bernoulli_posterior(success_history: List[bool],
+                             alpha: float = 0.5,
+                             beta: float = 0.5) -> float:
+    """See https://gregorygundersen.com/blog/2020/08/19/bernoulli-beta/"""
```

Comment:
i don't think i comprehend this even after reading the blog post: might want to ask you some questions in person